connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/util/RemoteSparkSession.scala

```diff
@@ -58,10 +58,12 @@ object SparkConnectServerUtils {
 
   private lazy val sparkConnect: Process = {
     debug("Starting the Spark Connect Server...")
-    val jar = findJar(
+    val connectJar = findJar(
       "connector/connect/server",
       "spark-connect-assembly",
       "spark-connect").getCanonicalPath
+    val catalystTestJar =
+      findJar("sql/catalyst", "spark-catalyst", "spark-catalyst", test = true).getCanonicalPath
```

Comment:
This would not work with a SBT only build, right?
Does this mean, for all SBT users, they have to build with maven once? Can you make sure the following case works:
```
build/sbt clean
build/sbt "testOnly org.apache.spark.sql.ClientE2ETestSuite”
```

Can you point me which is the test jar built by SBT?

Reply:
I try build with SBT locally. But I can't find where is test jar in target/scala-2.13. Should I change SBT build logic to add `sbt test:package`?

Reply:
good advise. done

Reply:
good advise. done